radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_PASTE                       "Вставити"
+#define TR_PASTE_AFTER                 "Вставити після"
+#define TR_PASTE_BEFORE                "Вставити до"
+#define TR_DELETE                      "Видалити"
+#define TR_INSERT                      "Вставити"
+#define TR_RESET_FLIGHT                "Скинути сесію"
+#define TR_RESET_TIMER1                "Скинути таймер1"
+#define TR_RESET_TIMER2                "Скинути таймер2"
+#define TR_RESET_TIMER3                "Скинути таймер3"
+#define TR_RESET_TELEMETRY             "Скинути телеметрію"
+#define TR_STATISTICS                  "Статистика"
+#define TR_ABOUT_US                    "Про..."
+#define TR_USB_JOYSTICK                "USB Ждойстик (HID)"
+#define TR_USB_MASS_STORAGE            "USB Сховище (SD)"
+#define TR_USB_SERIAL                  "USB Серійний (VCP)"
+#define TR_SETUP_SCREENS               "Налаштування екранів"
+#define TR_MONITOR_SCREENS             "Монітори"
+#define TR_AND_SWITCH                  "AND switch"
+#define TR_SF                          "SF"
+#define TR_GF                          "GF"
+#define TR_ANADIAGS_CALIB              "Калібровані аналоги"
+#define TR_ANADIAGS_FILTRAWDEV         "Фільтровані сирі аналоги з відхиленням"
+#define TR_ANADIAGS_UNFILTRAW          "Неформатовані сирі аналоги"
+#define TR_ANADIAGS_MINMAX             "Min., max. і діапазон"
+#define TR_ANADIAGS_MOVE               "Перемістити аналоги до екстремумів!"
+#define TR_SPEAKER                     INDENT "Гучномовець"
+#define TR_BUZZER                      INDENT "Дзвоник"
+#define TR_BYTES                       "бійти"
+#define TR_MODULE_BIND                 BUTTON(TR("Скр", "Скріпити"))
+#define TR_POWERMETER_ATTN_NEEDED      "Потрібен атенюатор"
+#define TR_PXX2_SELECT_RX              "Оберіть RX"
+#define TR_PXX2_DEFAULT                "<замовч>"
+#define TR_BT_SELECT_DEVICE            "Оберіть пристрій"
+#define TR_DISCOVER                    "Віднайти"
+#define TR_BUTTON_INIT                 BUTTON("Запуск")
+#define TR_WAITING                     "Очікуємо..."
+#define TR_RECEIVER_DELETE             "Видатити приймач?"
+#define TR_RECEIVER_RESET              "Скинути приймач?"
+#define TR_SHARE                       "Поділитися"
+#define TR_BIND                        "Скріпити"
+#define TR_REGISTER                    TR("Черв.", "Регістр")
+#define TR_MODULE_RANGE                BUTTON(TR("Діап.", "Діапазон"))
+#define TR_RANGE_TEST                  "Тест діапазону"
+#define TR_RECEIVER_OPTIONS            TR("ОПЦІЇ ПРМ.", "ОПЦІЇ ПРИЙМАЧА")
+#define TR_RESET_BTN                   BUTTON("Скидання")
+#define TR_DEBUG                       "Debug"
+#define TR_KEYS_BTN                    BUTTON("Кнопки")
+#define TR_ANALOGS_BTN                 BUTTON(TR("Анл.", "Аналоги"))
+#define TR_FS_BTN                      BUTTON(TR("Користув.перемик.", TR_FUNCTION_SWITCHES))
+#define TR_TOUCH_NOTFOUND              "Touch hardware не знайдено"
```

Comment:
These words could be translated as well
```suggestion
#define TR_TOUCH_NOTFOUND              "Сенсорний екран не знайдено"
```